cli/templatedelete.go

```diff
@@ -57,17 +67,19 @@ func templateDelete() *cobra.Command {
 				for _, template := range allTemplates {
 					if template.Name == selection {
 						templates = append(templates, template)
+						templateNames = append(templateNames, template.Name)
 					}
 				}
 			}
 
-			for _, templateName := range templateNames {
-				template, err := client.TemplateByName(ctx, organization.ID, templateName)
-				if err != nil {
-					return xerrors.Errorf("get template by name: %w", err)
-				}
-
-				templates = append(templates, template)
+			// Confirm deletion of the template.
+			_, err = cliui.Prompt(cmd, cliui.PromptOptions{
```

Comment:
I assume a test will have to be changed here too.

Reply:
done